Automattic/wp-calypso
# Earn: Update memberships to localized moment

## Proposed Changes

Updates the Earn > Memberships component to use the useLocalizedMoment hook rather than using moment directly. This is a small direct follow up to https://github.com/Automattic/wp-calypso/pull/80776

## Testing Instructions



There should be no change in functionality before/after this PR. 

1) Load this branch and go to the Earn > Memberships page at `http://calypso.localhost:3000/earn/payments/YOURDOMAIN`.
  
2) Test various things and confirm the look and behavior on this page is the same as it was before. (You can always compare by opening `https://wordpress.com/earn/payments/YOURDOMAIN` in a separate tab.)

3) Consider testing with and without a Stripe connectio. 

## Pre-merge Checklist
